twoliter/build.rs

```diff
@@ -20,12 +20,14 @@ fn main() {
     let paths = Paths::new();
     println!("cargo:rerun-if-changed={}", paths.data_input_dir.display());
 
+    let _ = fs::remove_dir_all(&paths.prep_dir);
```

Comment:
If we can avoid updating build.rs, that would be great. The entire file will be changed in an upcoming pr.

Reply:
> If we can avoid updating build.rs, that would be great. The entire file will be changed in an upcoming pr.

The addition of a file appears to be necessary for this PR, that is `Dockerfile.dockerignore`, so at least one change in `build.rs` is not optional. Anyway, the changes here should be easy to replay. Delete the prep dir and add a file.